src/ductile/document.clj

```diff
@@ -152,6 +152,15 @@
 (s/defschema BulkActions
   {BulkOps [(s/pred map?)]})
 
+(defn format-bulk-res
+  [bulk-res-list]
+  (reduce (fn [res elem]
+            {:took (+ (:took res 0) (:took elem))
+             :errors (and (:errors res true) (:errors elem))
```

Comment:
The `true` looks strange as it will cause most `and`s to succeed

Reply:
it's a weird trick that I need to comment. 
In this formatting function of ES results, at the init there is no `:errors` and I want to take the value of elem (which works with `and`).

Reply:
indeed it was not only a weird trick but also a bug :-)
I fixed it and added a dedicated unit test to not only rely on calling function test.